CCDB/src/ConditionsMQServer.cxx

```diff
@@ -9,23 +9,30 @@
  * ConditionsMQServer.cxx
```

Comment:
I still need to understand how much the CnditionsMQServer is tailored to the old OCDB case. The general server should not depend on these things. If the current implementation is a specialization for the OCDB case we have to think about a different name.